### ISSUE TITLE
支持 eslint 规则 space-before-blocks 

### ISSUE THREAD

**Issue by ShineKidd** [vuejs/eslint-plugin-vue#1621]
**Please describe what the rule should do:**

见 eslint 官方规则 https://eslint.org/docs/rules/space-before-blocks

**What category should the rule belong to?**

[ ] Enforces code style (layout)

**Provide 2-3 code examples that this rule should warn about:**

```vue
// bad
if (falg){ ... }

// good
if (flag) { ... }
```

**Additional context**

**Comment by ota-meshi** [comment]
I don't know the language you use, but I think that rule works fine.
